source/scale/scaling-for-enterprise.rst

```diff
@@ -23,6 +23,10 @@ The following reference architectures are available as recommended starting poin
 * :doc:`Scale up to 90000 users </scale/scale-to-90000-users>` - Learn how to scale Mattermost to up to 90000 users.
 * :doc:`Scale up to 100000 users </scale/scale-to-100000-users>` - Learn how to scale Mattermost to up to 100000 users.
 
+.. important::
+
+    Due to constraints in testing, both the proxy instance and the Elasticsearch cluster specifications were fixed for all the tests from which we derived these reference architectures. This was done to avoid a combinatorial explosion of variables in tests, but it resulted in minor gaps in our understandings of certain aspects of the reference architectures. In particular, the proxy instance is overspecified for the smaller user counts, while the Elasticsearch cluster is underspecified for user counts of 30k users or more.
```

Comment:
This is not true anymore for the Elasticsearch cluster. It should be something like:
```suggestion
    Due to constraints in testing, the proxy instance specifications were fixed for all the tests from which we derived these reference architectures. This was done to avoid a combinatorial explosion of variables in tests, but it resulted in minor gaps in our understandings of certain aspects of the reference architectures. In particular, the proxy instance is overspecified for the smaller user counts.
```